src/rust/ensogl/src/application/shortcut.rs

```diff
@@ -5,9 +5,14 @@ use crate::prelude::*;
 use super::command;
 
 use crate::control::io::keyboard::listener::KeyboardFrpBindings;
-use crate::frp::io::keyboard::Keyboard;
 use crate::frp::io::keyboard::KeyMask;
+use crate::frp::io::keyboard::Keyboard;
 use crate::frp;
+#[cfg(target_arch = "wasm32")]
+use crate::system::web;
+
+#[cfg(not(target_arch = "wasm32"))]
+use std::time;
```

Comment:
I dont like such conditional compilation. Instead, the `crate::system::*` should work across all platforms. Sure, it panics now on native, but its better to leave it in this stage and improve `crate::system::*` in the future than conditionally hide methods on usage side.